### PR TITLE
update docs/Contributing.md #782

### PR DESCRIPTION
**Pull request by marti1125**
update Contributing, adding some instructions #782

---
This change is [Reviewable](https://reviewable.io/reviews/cayleygraph/cayley/836)

### CODE REVIEW

File: docs/Contributing.md [hunk 1]
```diff
@@ -14,9 +22,13 @@ go mod download
 
 # build the binary
 go build ./cmd/cayley
+
+# try generated bin
```
Comment (dennwc) [inline docs/Contributing.md]:
```suggestion
# try the generated binary
```

File: docs/Contributing.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ Join our community on [discourse.cayley.io](https://discourse.cayley.io) or othe
 
 # Simply building Cayley
 
+If your version of Go < 1.13, you have to run:
+
+```
+export GO111MODULE=on
+```
+
+Follow the instructions for run locally
```
Comment (dennwc) [inline docs/Contributing.md]:
```suggestion
Follow the instructions for running Cayley locally:
```

File: docs/Contributing.md [hunk 3]
```diff
@@ -26,6 +38,9 @@ To run the web frontend, replace the "repl" command with "http"
 ./cayley http -i data/testdata.nq
 ```
 
+Open in your browser [cayley](http://127.0.0.1:64210)
```
Comment (dennwc) [inline docs/Contributing.md]:
```suggestion
You can now open the WebUI in your browser: http://127.0.0.1:64210
```

**Comment by marti1125** [comment]
In README.md I just added ` * Involvement: [Contribute](./docs/Contributing.md)`

**Comment by marti1125** [comment]
Great! about the section hacking, does it need change?

File: docs/Contributing.md [hunk 4]
```diff
@@ -70,6 +85,12 @@ Which will also resolve the relevant static content paths for serving HTTP.
 
 # Running Unit Tests
 
+If your version of Go < 1.13, you have to run:
```
Comment (dennwc) [inline docs/Contributing.md]: Yes, you are right, let's move this to the beginning of the "Hacking" section. E.g.:
```
First, you'll need Go [(version 1.11.x or greater)](https://golang.org/doc/install) ...

If your version of Go < 1.13, you have to run:
```

Also, it looks like the whole section needs an update regarding Go modules. But that's for a separate PR.

**Comment by marti1125** [comment]
:+1:  thanks

**Pull request merged**